src/s3fs.cpp

```diff
@@ -73,6 +73,7 @@ static bool is_mp_umask           = false;// default does not set.
 static std::string mountpoint;
 static std::string passwd_file;
 static std::string mimetype_file;
+//static std::string tenancy_id;
```

Comment:
Why is this commented out?

Reply:
Moved to globals. Removed in the latest commit.